evap/staff/views.py

```diff
@@ -1706,7 +1706,25 @@ def template_edit(request, template_id):
         messages.success(request, _("Successfully updated template."))
         return redirect('staff:index')
 
-    return render(request, "staff_template_form.html", dict(form=form, template=template))
+    available_variables = [
+        "contact_email",
+        "page_url",
+        "login_url",        # only if they need it
+        "user"
+    ]
+
+    if template.name == EmailTemplate.STUDENT_REMINDER:
+        available_variables += ["first_due_in_days", "due_evaluations"]
+    elif template.name == EmailTemplate.PUBLISHING_NOTICE_CONTRIBUTOR or template.name == EmailTemplate.PUBLISHING_NOTICE_CONTRIBUTOR or template.name == EmailTemplate.EDITOR_REVIEW_NOTICE or template.name == EmailTemplate.EDITOR_REVIEW_REMINDER:
```

Comment:
this has `EmailTemplate.PUBLISHING_NOTICE_CONTRIBUTOR` twice. Should be `_PARTICIPANT` for one of them?

Have you considered putting the elements in a list like this?
`elif template.name in [EmailTemplate.PUBLISHING_NOTICE_CONTRIBUTOR, ...]`

Reply:
Yes! Thanks for catching this error. I will use `in [...]` as this makes the code more readable.